database/src/test/scala/za/co/absa/atum/database/runs/CreatePartitioningIfNotExists.scala

```diff
@@ -0,0 +1,192 @@
+/*
+ * Copyright 2021 ABSA Group Limited
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package za.co.absa.atum.database.runs
+
+import za.co.absa.balta.DBTestSuite
+import za.co.absa.balta.classes.JsonBString
+
+class CreatePartitioningIfNotExists extends DBTestSuite{
+
+  private val fncCreatePartitioningIfNotExists = "runs.create_partitioning_if_not_exists"
+
+  private val partitioning = JsonBString(
+    """
+    |{
+    |  "version": 1,
+    |  "keys": ["key1", "key3", "key2", "key4"],
+    |  "values": {
```

Comment:
it might be a good idea to have it as per our agreement, especially because `create_partitioning_if_not_exists` will call those functions for partitioning validation: https://github.com/AbsaOSS/atum-service/pull/109

```suggestion
    |  "keysToValuesMap": {
```

Reply:
also on all other places in this PR